changelog/_unreleased/2024-12-04-improve-sw-page-back-button-linking.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Improve sw-page back button linking
+issue: NEXT-00000
```

Comment:
```suggestion
```

Reply:
Done, but now the lint:changelog fails 😅

Reply:
Oh I see 🙈 I'll challenge that internally how we want to handle that now that we are completely on GitHub 😊 
CC: @pweyck

Reply:
It should also work if you reference a gh issue id instead of a next reference to the changelog file or add a `fixes #<issue>` to the commit message with the changelog file. 

But if we only have PR it should also be ok, we'll just link the PR. I'll fix that soon.